Framework/PythonInterface/mantid/kernel/src/kernel.cpp.in

```diff
@@ -51,6 +51,10 @@ BOOST_PYTHON_MODULE(_kernel) {
   Poco::LoggingFactory::defaultFactory().registerChannelClass(
     "PythonStdoutChannel",
     new Poco::Instantiator<Poco::PythonStdoutChannel, Poco::Channel>);
+  /* // Store in case class PythonStdoutChannel is not up to the task and needs to be replaced with this class
```

Comment:
A comment?  Applied anywhere?

Reply:
I replaced with the same TODO comment as before